cmd/service-controller/controller.go

```diff
@@ -10,6 +10,8 @@ import (
 	"strings"
 	"time"
 
+	clientpodman "github.com/skupperproject/skupper/client/podman"
```

Comment:
I wonder if it might also be clearer to keep these in a distinct file. The current controller does define far too much stuff in the main package. We should move stuff into explicit packages if needed.

Reply:
If we keep a separate package/image to the podman controller, the new `ControllerPodman` could be placed
there and the `collector_beacons.go` could be moved under `pkg/domain/podman` as already suggested by
@ajssmith.

I will separate it as a new process and image, then we can reevaluate it.